GWAS/PCA.ipynb

```diff
@@ -985,17 +991,17 @@
     "    colnames(pca) <- c(\"ID\",paste0(\"PC\", 1:${k}))\n",
     "    pca$IID <- sapply(strsplit(as.character(pca$ID),':'), \"[\", 1)\n",
     "    # Read fam file with phenotypes\n",
-    "    pheno <- read.table(${_input[1]:r}, sep=\"\\t\", header=T,stringsAsFactors =F)\n",
+    "    pheno <- read.table(${_input[1]:r}, sep=\"\\t\", header=F,stringsAsFactors =F)\n",
```

Comment:
We have to make this an option and not simply set header to FALSE

Reply:
done